ci/staging-acceptance-tests.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+
+set -uo pipefail
+
+# These are destructive acceptance tests which should only be run in STAGING, there are explicit checks for this, do not override them
+# Requires a user with cloud_controller.admin access to run since it will be creating/deleting users, organizations and org quotas
+# These are destructive to the `sandbox-fedramp`.  This org was chosen since it: wasn't in use, we have email access to the domain, should remain in the CSV file maintaining the list of verified gov agencies
+
+# Log into CF as an admin
+echo "Logging into ${CF_API} as user ${CF_ADMIN_USER}..."
+cf login -a ${CF_API} -u ${CF_ADMIN_USER} -p "${CF_ADMIN_PASSWORD}" -o cloud-gov -s bots >/dev/null 2>&1
+
+# Confirm that we're targeting the correct API
+api_target=$(cf api | grep -i 'API endpoint' | awk '{print $3}')
+if [[ "$api_target" != *"fr-stage"* ]]; then
+  echo "### THIS IS A DESTRUCTIVE TEST to the sandbox-fedramp org, DO NOT RUN IN PRODUCTION ###"
+  echo "Error: Not targeting staging. Current API endpoint: $api_target, exiting for your own safety."
+  exit 1
+fi
+
+# Create a random 32-character password with hyphens
+PASSWORD=$(cat /dev/urandom | base64 | tr -dc '0-9a-zA-Z' | head -c32)
+
+# Create the user
+echo "Creating CF user test.user@fedramp.gov with a random password..."
+cf create-user "test.user@fedramp.gov" "$PASSWORD" >/dev/null 2>&1
```

Comment:
This is very much personal preference, but I would move these lines right above the other lines where you're creating the second test user because these commands don't matter in the tests until we start checking for expected spaces

Reply:
I'm looking to have this done as two separate runs of the sandbox-bot so I can verify that the "update" versus the "Create" for the org quota.

Reply:
Oh, I see. Because creating the test user triggers the creation of the sandbox org and its quota. Gotcha